### ISSUE TITLE
Suffix in the urlManager makes Admin Panel broken

### ISSUE THREAD

**Issue by phplego** [noumo/easyii#65]
When there is this option configured:
```php
<?php
....

'urlManager'=>[
            'suffix'=>'/',
            ...
]
```

It makes some javascript-built urls little wrong. For example:
```js
location.href = checkbox.attr('data-link') + '/' + (checkbox.is(':checked') ? 1 : 0);
```
In the frontend.js

**Comment by phplego** [comment]
The only stable solution I see is don't build the urls manually in the JS.
In the old Yii we did it with something like this:
```php
Yii::app()->clientScript->registerScript('helpers', '
        someUrl = '.CJSON::encode(Yii::app()->createUrl('blabla')).';
        baseUrl = '.CJSON::encode(Yii::app()->baseUrl).';
');?>
```

**Comment by phplego** [comment]
The solution is to use "url templates":
```php
'data-link' => Url::to(['/admin/'.$module.'/items', 'value=>'@value']),
```
It creates url something like this:
```
http://yousite.com/admin/books/items/@value/
```

Then we able to replace @value to real value:
```js
location.href = checkbox.attr('data-link').replace('@value',  checked ? 1 : 0));
```

**Comment by noumo** [comment]
Can you make a pull request?

**Comment by phplego** [comment]
I was trying to implement this solution but have faced with a problem.  The problem is that it requires to change admin rotes (because pattern \d+ do not allow to use "@id" as value). And I'm not sure if this change affect other parts of code.

**Comment by noumo** [comment]
Later i will try to split application on backend and frontend like yii2 advanced template and then it will be possible to apply different configs on back and front.

**Comment by phplego** [comment]
Oh this division on two parts is one of main reasons why usual developers avoid yii2. Just because too much configs, too much almost same folders, too much unused flexibility, and it requires to configure web server document root to subfolder and make it twice: for frontend and backend. For normal developer who don't fap on "super-security" it brings nothing but pain in the ass.

In my view it is resonable to find some intermediate solution, when we have different config for admin area and frontend area but do not have all that mess of options as in yii2-advanced. Thanks for reading

**Comment by noumo** [comment]
I agree with you to some extent about spliting application, thats why easyii examples is very simple.
If you implement such solution i will be glad to recieve pull request.
